M2/Macaulay2/packages/Graphs.m2

```diff
@@ -3434,7 +3432,6 @@ doc ///
 --diameter
 doc ///
     Key
-        diameter
```

Comment:
We probably have to also add some way for users of "Graphs" to discover this documentation node, by linking to it from another suitable one.

Reply:
There were three other nodes that pointed to `diameter` under `SeeAlso`, so I've just pushed another commit updating those to `(diameter, Graph)`.

It's also accessible under the main `Graphs` page, plus the index and table of contents.